data/governments.txt

```diff
@@ -320,6 +320,14 @@ government "Kor Mereti"
 		"Kor Sestor" -.01
 		"Wanderer" -.01
 	
+	"penalty for"
+		assist 0
+		disable 0
+		board 0.5
+		capture 5
+		destroy 5
+		atrocity 0
```

Comment:
Why the changes to the penalties? And why a penalty of 5 for destroying or capturing? You might as well just keep the capture and destroy penalties at their default and have the crew equivalent values be 5 times what they are now.

Reply:
These penalties should actually start at 0 then be changed to the default when the Mereti become friendly.

Reply:
> These penalties should actually probably start at 0 then be changed to the default when the Mereti become friendly. The Sestor could also have their penalties be 0.

The thing is then we have to work with events which I don't really like, it doesn't hurt to have these penalties always be 5,, considering nobody cares about them enough for it to change your reputation, you just get lower rep with Mereti and Sestor which doesn't matter at all.
This way when smbd adds a way to make Sestor friendly for example, they won't have to create an event for it too. Also the Wanderer campaign could have the Mereti be hacked once again and turning back against you, it seems to me the simplest option just is to have it be this way and not touch it.